### PR TITLE
Issue #4933: Unable to save node after using CKEditor "Maximize"

### LINKED ISSUE #4933: Fix CKEditor "Maximize" mode: form should save successfully after closing
**Description of the bug**

Sometimes I'm not able to save a page after editing the body text in CKEditor. After hitting the "Save" button of the page edit form, the editor area changes: buttons disappear,  text is in source code, and the whole area seems frozen. To continue editing, I have to reload the edit form, and my edits of before are lost!

The issue doesn't happen all the times but quite often. I've experienced it on many different sites over a long time. (Didn't report it before, because I'm not sure how to exactly reproduce it reliably.)

**Steps To Reproduce**

To reproduce the behavior:
1. Add a page with some content, and save it.
2. Edit the page again. Maximize the CKEditor area (the "Maximize" button stays usually at the right of the toolbar).
3. Do some editing within the textarea. Insert also an image.
4. Minimize the CKEditor area again. (Everything fine so far.)
5. Hit the Save button of the content form.

**Actual behavior**

*Sometimes* the CKEditor area looses its buttons and seems frozen. Within the area, you see the source code of your content. The content isn't editable, and the content form doesn't seem to react in any way. The content doesn't get saved, and you just stay in the form. To continue, you have to hit the "Edit" tab at the top of the form and make your content changes again.

**Expected behavior**

The page is saved, and you are directed to where you came from.

**Additional information**

- Backdrop CMS version: various (not only the latest)
- Web server: various
- Operating System and Browsers: *most* of the times, I use Firefox on Mac OS

### PR DESCRIPTION
**Pull request by indigoxela**
Fixes https://github.com/backdrop/backdrop-issues/issues/4933

### CODE REVIEW

File: core/misc/form.js [hunk 1]
```diff
@@ -98,6 +98,8 @@ Backdrop.behaviors.formUpdated = {
 Backdrop.behaviors.formSingleSubmit = {
   attach: function () {
     function onFormSubmit (e) {
+      // Prevent this from fireing multiple times per request.
```
Comment (klonos) [inline core/misc/form.js]: Small typo:

```suggestion
      // Prevent this from firing multiple times per request.
```

Comment (indigoxela) [reply core/misc/form.js]: Hm. Are my translation tools tricking me? :shrug: They still suggest "fireing".

Comment (klonos) [reply core/misc/form.js]: 🤷 

image

image

https://www.spellchecker.net/misspellings/fireing

> This word (Fireing) may be misspelled. Below you can find the suggested words which we believe are the correct spellings for what you were searching for. If you click on the links, you can find more information about these words.

Comment (indigoxela) [reply core/misc/form.js]: :smile: already updated, anyway... :+1:

**Comment by quicksketch** [comment]
Merged into 1.x and 1.18.x.

**Pull request merged**